samples/client/petstore/dart-dio/petstore_client_lib/lib/api/pet_api.dart

```diff
@@ -18,7 +18,7 @@ class PetApi {
     /// Add a new pet to the store
     ///
     /// 
-    Future<Response>addPet(
+    Future<Response<void>> addPet(
```

Comment:
are you sure this is correct for adding a pet? persumably it's for POST and `_dio.request` has `bodyData`.

Why is uploading file different than uploading object Pet?

Reply:
No response defined here: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing.yaml#L42

And 200 with ApiResponse here: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore-with-fake-endpoints-models-for-testing.yaml#L259 200

Reply:
Btw this is the response, the `bodyData` doesn't matter.